### PR TITLE
Parameter list table #87

### PR DESCRIPTION
**Pull request by sunny-github-acc**

### CODE REVIEW

**Comment by saulyz** [comment]
is your IDE under proper configuration? Or do you use something custom, not compatible with project recommendations? Did you remove prettier from `eslintrc.json`?

File: src/components/animal/events/AnimalEventDialog.tsx [hunk 1]
```diff
@@ -0,0 +1,282 @@
+import React, { useState } from 'react';
```
Comment (saulyz) [inline src/components/animal/events/AnimalEventDialog.tsx:1]: what is this file about in the ParamTable PR? Looks like the changes are irrelevant.

Comment (sunny-github-acc) [reply src/components/animal/events/AnimalEventDialog.tsx]: Indeed it was irrelevant. Already removed it.
As far as I can tell, I have not used anything custom. Have not deleted prettier from eslintrc.json neither.

**Comment by saulyz** [comment]
Thank you we have this feature in the codebase. We will close this PR.

**Pull request closed without merging**